NuGetCleaner/Program.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Reflection;
+using System.IO;
+using CommandLine;
+using System.ComponentModel;
+
+namespace NuGetCleaner
+{
+    class Program
+    {
+
+        public static void Main(string[] args)
+        {
+            var osNameAndVersion = System.Runtime.InteropServices.RuntimeInformation.OSDescription;
+            if (!osNameAndVersion.Contains("Windows"))
+            {
+                Console.WriteLine("This tool is currently only available for Windows, sorry for the inconvenience!\n");
+                return;
+            }
+
+            if (CheckDisableLastAccess() != 2)
+            {
+                Console.Write("\nYour Last Access updates are not currently enabled so this tool will not work. \n" +
+                    "To enable Last Access updates, run powershell as administrator and input:\n\n" +
+                    "\tfsutil behavior set disablelastaccess 2\n\n" +
+                    "You may be asked to reboot for the settings change to take effect\n\n" +
+                    "Note: Last usage time of packages will only be tracked once the setting is enabled.\n" +
+                    "As such, last package usage will only be determined from the enable date onward.\n" +
+                    "For further information view documentation at https://github.com/chgill-MSFT/NuGetCleaner \n");
+                return;
+            }
+
+            Parser.Default.ParseArguments<Options>(args)
+                .WithParsed(options => Execute(options));
+        }
+
+        public static void Execute(Options options)
+        {
+
+            string Path;
+
+            if (options.Path == null)
+            {
+                Path = "C:\\Users\\" + Environment.UserName + "\\.nuget\\packages";
```

Comment:
Could we use client APIs to discover this?

Reply:
Also, `Path.Combine` is more idiomatic for path construction.

IIRC, there is a .NET API or environment variable to get the user directory so you don't have to hard-code to `C:\` drive.

Reply:
@joelverhagen could you elaborate on the usage of client APIs? Do you mean that there is probably a client team API for automatically finding the global package folder?